src/components/application/appsco-application-group-remove.html

```diff
@@ -0,0 +1,204 @@
+                    type: String,
+                    value: ''
+                },
+
+                _headers: {
+                    type: String,
+                    computed: '_computeHeaders(authorizationToken)'
+                },
+
+                _loader: {
+                    type: Boolean,
+                    value: false
+                },
+
+                /**
+                 * Error message.
+                 */
+                _error: {
+                    type: String
+                }
+            },
+
+            setGroup: function(group) {
+                this.group = group;
+            },
+
+            open: function () {
+                this.$.dialog.open();
+            },
+
+            close: function () {
+                this.$.dialog.close();
+            },
+
+            toggle: function () {
+                this.$.dialog.toggle();
+            },
+
+            _computeHeaders: function(authorizationToken) {
+                return {
+                    'Authorization': 'token ' + authorizationToken,
+                    'Content-Type': 'application/x-www-form-urlencoded'
+                };
+            },
+
+            _computeRemoveGroupApi: function(group) {
+                return group.meta ? group.meta + '/applications/' + application.id : '';
+            },
+
+            _showLoader: function() {
+                this._loader = true;
+            },
+
+            _hideLoader: function() {
+                this._loader = false;
+            },
+
+            _showError: function(message) {
+                this._error = message;
+            },
+
+            _hideError: function() {
+                this._error = '';
+            },
+
+            _onDialogClosed: function() {
+                this._hideError();
+                this._hideLoader();
+            },
+
+            _onRemove: function() {
+                var request = document.createElement('iron-request'),
+                        options = {
+                            url: this._removeGroupApi,
+                            method: 'DELETE',
+                            handleAs: 'json',
+                            headers: this._headers
+                        };
+
+                this._showLoader();
+
+                request.send(options).then(function() {
+                    this.close();
+
+                    if (200 === request.status) {
+                        var group = request.response;
+
+                        /**
+                         * Fired when group is deleted.
+                         *
+                         * @event group-removed
+                         * @param [Object] group
+                         */
+                        this.fire('group-removed', {
+                            group: group
+                        });
+                    }
+
+                }.bind(this), function() {
+
+                    if (request.status === 404) {
```

Comment:
Here you should use appsco-api-errors to display error message.
Check **src/directory/appsco-directory-add-account.html** for example.